crits/objects/object_mapper.py

```diff
@@ -322,20 +322,20 @@ def make_crits_object(cybox_obj):
     o.datatype = "string"
     if isinstance(cybox_obj, Account):
         o.object_type = "Account"
-        o.value = str(cybox_obj.description)
+        o.value = cybox_obj.description.values
         return o
     elif isinstance(cybox_obj, Address):
         o.object_type = "Address"
         o.name = str(cybox_obj.category)
-        o.value = str(cybox_obj.address_value)
+        o.value = cybox_obj.address_value.values
         return o
     elif isinstance(cybox_obj, API):
         o.object_type = "API"
-        o.value = str(cybox_obj.description)
+        o.value = cybox_obj.description.values
```

Comment:
I was doing some testing and encountered an error:
`API: 'StructuredText' object has no attribute 'values'`
It turns out that the description field of CybOX objects like "API", "Code", and "DNS_Record" (I'm sure there are more) are based on class StructuredTest which doesn't have a "values" property. Changing "values" to "value" seems to help, but I'm not certain that is the correct answer.